test/SpecRunner.html

```diff
@@ -24,6 +25,7 @@
 
   <!-- include source files here... -->
   <script src="../../fast-json-patch/src/json-patch-duplex.js"></script>
+  <script src="../../../node_modules/jsonpatcherproxy/dist/jsonpatcherproxy.min.js"></script>
   <script src="../src/palindrom.js"></script>
   <script src="../src/palindrom-dom.js"></script>
 
```

Comment:
I noticed this too. But `fast-json-patch` is coming from `bower_components` dir, thanks to Polyserve for handling this weird looking request. That's way Palidrom tests almost ruined my day two months ago when I tried to run them in WAMP web server instead of Polyserve.

However, `SpecRunnerBundle.html` is clean and is using Palindrom bundle as a single dependency.